.gitignore

```diff
@@ -14,6 +14,8 @@ __pycache__/
 
 .pytest_cache
 
+test/test_db.sqlite/
```

Comment:
This isn't working because of the `/` at the end. I see `test/test_db.sqlite` has still been pushed.

Reply:
He should actually delete it because it's already taken care of by `*.sqlite`

Reply:
@Ashaba I am going to do for you this, so that I merge this PR